src/IO/HDF5/ParallelHDF5IOHandler.cpp

```diff
@@ -70,8 +70,10 @@ ParallelHDF5IOHandlerImpl::ParallelHDF5IOHandlerImpl(
         hsize_t page_size;
         tstream >> page_size;
 
+#if H5_VERSION_GE(1,10,1)
```

Comment:
Do you want to move this even up to guard the whole block? (line 66-77)

Reply:
Sure, makes sense!